.env.example

```diff
@@ -0,0 +1,10 @@
+CF_API_KEY=
+CF_AUTH_EMAIL=
+CF_ROOT_DOMAIN=
+SERVICE=
+RECORD=
+TTL=
+CB_USER=
```

Comment:
Two points worth considering:

1. The Couchbase container expects `COUCHBASE_USER`. 
2. `CB` seems ambiguous, especially since we launched Containerbuddy.